erc20-holdings/substreams.yaml

```diff
@@ -22,7 +22,7 @@ binaries:
 modules:
   - name: map_block_to_erc20_contracts
     kind: map
-    initialBlock: 10606500
+    initialBlock: 1
```

Comment:
Slight misalignment with this initial block and the make run cmd (cmd starts at 0 rather than 1) - perhaps you can change one or the other to make both line up?

Reply:
I updated the cmd to start at 1, before it didn't have any start block so it just started at 1 by default